pyls/workspace.py

```diff
@@ -237,15 +230,16 @@ def jedi_script(self, position=None):
         environment = self.get_enviroment(environment_path) if environment_path else None
 
         kwargs = {
-            'source': self.source,
+            'code': self.source,
             'path': self.path,
-            'sys_path': sys_path,
             'environment': environment,
+            'project': jedi.api.Project(os.path.dirname(self.path),
```

Comment:
Should this use `self._workspace.root_path`? See #801.

Reply:
@ccordoba12 wants me to answer this question. But I think a PyLS developer should know this better than me. So should it? I don't know. What difference does it make? How is the internal _workspace attribute used in PyLS?

Reply:
After taking a look at this, I think @fsouza is right.

Please make this change @bnavigator.

Reply:
```suggestion
            'project': jedi.Project(path=project_path, sys_path=sys_path),
```